spring-boot3-dummy/pom.xml

```
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.axonframework</groupId>
+        <artifactId>axon</artifactId>
+        <version>4.9.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>axon-spring-boot3-dummy</artifactId>
+    <name>Spring Boot 3 compiler dummy</name>
```

Small rename suggestion to align with the other pom names:
```suggestion
    <name>Axon Framework - Spring Boot 3 Compiler Dummy</name>
```